tests/python/gaia-ui-tests/gaiatest/apps/bzlite/app.py

```diff
@@ -0,0 +1,72 @@
+from marionette import Marionette
+from gaiatest.apps.base import Base
+from marionette_driver import By, Wait, expected
+
+class BugzillaLite(Base):
+
+    name = 'Bugzilla Lite'
+    _given_username = (By.CSS_SELECTOR, "#login input[type='email']")
+    _given_password = (By.CSS_SELECTOR, "#login input[type='password']")
+    _button_login = (By.CSS_SELECTOR, "#login input[type='submit']")
+    _profile_icon = (By.CSS_SELECTOR, ".profile")
+    _button_logout = (By.CSS_SELECTOR, ".btn")
```

Comment:
Actually the Selector in the DOM is btn warning and I thought we can't use two Selector with By.CSS_SELECTOR, I searched in the CSS file now and I saw that the class .btn used in other elements and the class .warning too so I suggest to use the Xpath in place of CSS_SELECTOR! what do you think ?